### PR TITLE
utils : move panic handling to separate package

### PR DESCRIPTION
**Pull request by ramaraochavali**
Moves panic handling to separate package as it can create cyclic dependencies if we want to use to in model package in future.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ X] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ramaraochavali** [comment]
@hzxuzhonghu See if this change makes sense. Basically, If I try to use this in model package, I ran in to cyclic dependency. So proposing to move it out. But it is not needed now, immediately. If you think, it is not worth moving, I can close this PR.

**Comment by hzxuzhonghu** [comment]
We can do that, but how about renaming `panic` to `runtime`?

**Comment by ramaraochavali** [comment]
Initially I did the same - but it conflicted with go's runtime. Any better names?

**Comment by ramaraochavali** [comment]
@hzxuzhonghu sorry, Ignore what I said. I moved to `runtime`. PTAL

File: galley/pkg/crd/validation/validation.go [hunk 1]
```diff
@@ -146,6 +146,7 @@ func RunValidation(ready chan<- struct{}, stopCh chan struct{}, vc *WebhookParam
 			if readinessProbeController != nil {
 				validationReadinessProbe.SetAvailable(errors.New("stopped"))
 			}
+			// nolint: staticcheck
```
Comment (sushicw) [inline galley/pkg/crd/validation/validation.go]: What is this needed for?

Comment (howardjohn) [reply galley/pkg/crd/validation/validation.go]: should not be needed now that https://github.com/istio/istio/commit/3cb2c39d24a449896a778b5739512c8c122ede5b merged

**Comment by hzxuzhonghu** [comment]
@ramaraochavali please rebase

**Comment by ramaraochavali** [comment]
@hzxuzhonghu rebased

**Pull request merged**